Resources/Locale/en-US/pda/pda-component.ftl

```diff
@@ -41,3 +41,5 @@ pda-bound-user-interface-music-button-description = Play music on your PDA
 comp-pda-ui-unknown = Unknown
 
 comp-pda-ui-unassigned = Unassigned
+
+comp-pda-ui-station-alert-level = The current station alert level is: [color={ $ColorLevel }]{ $AlertLevel }[/color]
```

Comment:
Too complicated imo, just `Alert level: [color={ $ColorLevel }]{ $AlertLevel }[/color]` would be better

Reply:
i don't know. The title i get from emergence lamps.

Reply:
sorry i closed this PR because it's a duplicate of this one, all notes write there https://github.com/space-wizards/space-station-14/pull/15220